doc/source/whatsnew/v0.20.0.txt

```diff
@@ -122,6 +122,16 @@ Other enhancements
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. _whatsnew.api_breaking.io_compat
+
+Pickle and HDF5 compat requires >= 0.13.0
```

Comment:
This title is not fully clear. I would at least add 'requires *pandas* >= 0.13.0'. But you mean that pickle and HDF5 files created with pandas versions older than 0.13.0 may not be readable anymore?